doaj/DOAJ/DOAJRequest.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+#author:  Víctor Moreno Marín, Gerardo Gutiérrez
+#email: vmorenomarin@gmail.com, muzgash@gmail.com, 
+
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+from forex_python.converter import CurrencyRates
+import requests             
+import json   
+import numpy as np  
+
+
+class DOAJRequest:
+    '''
+    Create a DOAJ object to request and save data from DOAJ site using its API.
+    '''
+
+    def __init__(self,database_name,collection):
+        self.client = MongoClient()
+        self.db = self.client[database_name]
+        self.collection = self.db[collection]
+     
+    def getIssnLists(self): 
```

Comment:
@muzgash  all the ISSNs can be taken at the moment from lens, but this use case should be improved, how to handle if I need an issn from other db?